### PR TITLE
OSSMDOC-568: Service Mesh Z Stream Release Notes

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s):
4.6-4.11

Jira Issue: https://issues.redhat.com/browse/OSSMDOC-568

Link to rel notes 2.x docs preview: https://deploy-preview-45211--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html
Link to rel notes 1.x docs preview: https://deploy-preview-45211--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v1x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-1-1-18-1

  

Additional information:

### CODE REVIEW

**Review by rcernich — APPROVED** [review]
lgtm

File: modules/ossm-rn-new-features-1x.adoc [hunk 1]
```diff
@@ -19,6 +19,28 @@ Module included in the following assemblies:
...
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs).
+
+=== Component versions included in {SMProductName} version 1.1.18.1
+
+|===
+|Component |Version
+
+|Istio
+|1.4.8
```
Comment (jwendell) [inline modules/ossm-rn-new-features-1x.adoc:32]: It's 1.4.10 for a while, I think we never updated this in the docs.

**Review by jwendell — APPROVED** [review]

File: modules/ossm-rn-new-features-1x.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features-1x.adoc]: Thanks for pointing this out. The edit has been made.

File: modules/ossm-rn-fixed-issues.adoc [hunk 2]
```diff
@@ -19,9 +19,15 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-1099[OSSM-1099]
+Kiali shows `Sorry, there was a problem. Try a refresh or navigate to a different page.`
```
Comment (JStickler) [inline modules/ossm-rn-fixed-issues.adoc:23]: The Kiali console displayed a message, `Sorry, there was a problem. Try a refresh or navigate to a different page.`

**Review by JStickler — COMMENTED** [review]
Couple of small tweaks.  But looks good!

File: _attributes/common-attributes.adoc [hunk 3]
```diff
@@ -118,7 +118,7 @@ endif::[]
 :product-dedicated: Red Hat OpenShift Dedicated
 :SMProductName: Red Hat OpenShift Service Mesh
 :SMProductShortName: Service Mesh
-:SMProductVersion: 2.1.2
+:SMProductVersion: 2.1.2.1
 :MaistraVersion: 2.1
 //Service Mesh v1
 :SMProductVersion1x: 1.1.18
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply _attributes/common-attributes.adoc:124]: Oh, yes, I didn't initially realize that and never thought to go in and change this later. Thanks, @JStickler!

Done.

File: modules/ossm-rn-fixed-issues.adoc [hunk 4]
```diff
@@ -19,9 +19,15 @@ The following issues been resolved in the current release:
...
+
 * https://issues.redhat.com/browse/OSSM-1074[OSSM-1074]
 Pod annotations defined in SMCP are not injected in the pods.
 
+* https://issues.redhat.com/browse/OSSM-999[OSSM-999]
+Kiali retention does not work as expected. Calendar times are greyed out in the dashboard graph.
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-fixed-issues.adoc:29]: Done

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**